include/wally_address.h

```diff
@@ -47,6 +56,22 @@ WALLY_CORE_API int wally_addr_segwit_to_bytes(
     size_t len,
     size_t *written);
 
+/**
+ * Infer a scriptPubKey from an address.
+ *
+ * :param addr: Address to infer the scriptPubKey from.
+ * :param flags: Pass ``WALLY_NETWORK_BITCOIN_MAINNET`` or ``WALLY_NETWORK_BITCOIN_TESTNET``.
```

Comment:
nit: I'm not sure I would call these "flags" as they are mutually exclusive

Reply:
We use mutually exclusive flags for `EC_FLAG_ECDSA` and `EC_FLAG_SCHNORR` too.